cmd/policy/login.go

```diff
@@ -74,3 +82,33 @@ func (c *LoginCmd) Run(g *Globals) error {
 	<-g.App.Context.Done()
 	return nil
 }
+
+func checkDefault(g *Globals, c *LoginCmd) (bool, error) {
+	setDefault := false
+	if g.App.Configuration.DefaultDomain == "opcr.io" && c.Server != g.App.Configuration.DefaultDomain && !c.DefaultDomain {
+		g.App.UI.Normal().WithAskBoolMap("Do you want to set this server as your default domain?[yes/no]", &setDefault, map[string]bool{
+			"yes": true,
+			"no":  false,
+			"y":   true,
+			"n":   false,
+			"Y":   true,
+			"N":   false,
+		}).Do()
+	}
+	if c.DefaultDomain {
+		setDefault = true
+	}
+
+	// Reset all defaults to false to return true and set the current login as default
```

Comment:
// Reset all defaults to false to return true and set the current login as default
```suggestion
	// Reset all defaults to false to return true in order to set the current login as default
```